### PR TITLE
Update build/distribution tooling.

### PR DESCRIPTION
**Pull request by jgerigmeyer**
- [x] Compile TS to JS and distribute type defs
- [x] Bundle JS into ESM + UMD (CJS/IIFE) bundles
- [x] Transpile all code (including npm deps) down to es6 target browsers

### CODE REVIEW

File: src/fetching.ts [hunk 1]
```diff
@@ -22,5 +24,30 @@ export function transformCSS(positionFallbackRules: PositionFallbackRulesMap) {
...
+        position: strategy,
+        left: `${x}px`,
+        top: `${y}px`,
+      });
+    };
+
+    computePosition(referenceElement, floatingElement, {
+      placement: 'bottom',
+    }).then(applyStyles);
+  }
```
Comment (jgerigmeyer) [inline src/fetching.ts:52]: All this stuff can get removed/replaced with the actual polyfill code -- it's just to test that this package can be installed/run, including the floating-ui code.

File: vite.config.ts [hunk 2]
```diff
@@ -8,7 +9,9 @@ export default defineConfig({
       name: 'CssAnchorPositioning',
       // the proper extensions will be added
       fileName: 'css-anchor-positioning',
-      formats: ['es', 'umd', 'iife'],
     },
+    target: 'es6',
```
Comment (jgerigmeyer) [inline vite.config.ts:13]: @sanajaved7 @oluoluoxenfree Right now this is targeting es6, which is pretty straightforward. If we want to target es5 we might need to consider a different build tool (Babel?) -- Vite and esbuild [do not support](https://vitejs.dev/guide/build.html#browser-compatibility) [es5 transpilation yet](https://esbuild.github.io/content-types/#es5).

File: src/index.ts [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import { transformCSS } from './fetching.js';
 
 // Expose API
+// @@@ This should be replaced with the actual API we want to expose
```
Comment (jgerigmeyer) [inline src/index.ts:4]: Again, just for testing purposes. I imagine that the API we want to expose would go here.

**Review by jgerigmeyer — COMMENTED** [review]
@sanajaved7 @oluoluoxenfree @jerivas This is an initial pass, ready for review/feedback.

**Comment by jgerigmeyer** [comment]
[Refine build process](https://trello.com/c/sTQjtl80/39-refine-build-process)

File: vite.config.ts (hunk 2 above)
Comment (VicGUTT) [reply vite.config.ts:13]: ES2020+ 😎

**Review by sanajaved7 — APPROVED** [review]

**Comment by VicGUTT** [comment]
Yay this has been merged ! :)

But quick question, are folks able to see my comments other than "[ES2020+ 😎](https://github.com/oddbird/css-anchor-positioning/pull/10#discussion_r942684052)" ? 🤔 Where I tried to "start reviews" and provide some feedback to the implementation.

If not, please let me know, I'll try to re-post them or screenshot what I'm seeing.

**Comment by jerivas** [comment]
I only see the ES2020 comment 😕 Did you "submit" the review after adding inline comments (top right corner)?

**Comment by VicGUTT** [comment]
> I only see the ES2020 comment 😕 Did you "submit" the review after adding inline comments (top right corner)?
> 
> 

Wow good catch, thank you. I won't be making that mistake again ^^

**Comment by jgerigmeyer** [comment]
> But quick question, are folks able to see my comments other than "[ES2020+ 😎](https://github.com/oddbird/css-anchor-positioning/pull/10#discussion_r942684052)" ?

@VicGUTT I made some changes based on your suggestions -- please review https://github.com/oddbird/css-anchor-positioning/pull/12 when you are able. Thanks!

**Pull request merged**